lib/isolate_runner.dart

```diff
@@ -185,7 +184,8 @@ class IsolateRunner implements Runner {
   ///     } finally {
   ///       await iso.close();
   ///     }
-  Future run(function(argument), argument, {Duration timeout, onTimeout()}) {
```

Comment:
Not going to work...need to fix subclasses, too!

Might want to build on https://github.com/dart-lang/isolate/pull/11 – I enable strong mode checks, etc

Reply:
> Not going to work...need to fix subclasses, too!

Where do I find the subclasses that need to be fixed?

> Might want to build on #11 – I enable strong mode checks, etc

If you think #11 should land first, I have no problem with that. But I have no idea what "build on" means in this context.